### PR TITLE
Added tooltip to avatar component

### LINKED ISSUE #1308: Fix TimePicker color in dark mode
image

### PR DESCRIPTION
**Pull request by amaldinesh7**
Fixes #1308 

**Description**

- Added: tooltip to _Avatar_ component

**Checklist**

- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary label (patch/minor/major - If package publish is required)
- [x] I have followed the suggested description format and styling

**Reviewers**

### CODE REVIEW

**Comment by amaldinesh7** [comment]
@ajmaln _a Please review

File: stories/Components/Avatar.stories.jsx [hunk 1]
```diff
@@ -65,6 +65,24 @@ export const WithStatus = ({}) => {
...
+  return (
+    <div className="flex space-x-6 ">
+      <Avatar user={{ name: "Neeraj Singh" }} size="large" showTooltip />
+      <Avatar
+        user={{ name: "Amal Dinesh", imageUrl }}
+        size="large"
+        showTooltip
+      />
+
+      {/* <Avatar user={{}} status="idle" size="large" /> */}
```
Comment (ajmaln) [inline stories/Components/Avatar.stories.jsx:80]: Remove the comment

**Review by ajmaln — APPROVED** [review]
@amaldinesh7 _a Changes LGTM, a minor nitpick

**Comment by amaldinesh7** [comment]
@ajmaln _a Added unit test and resolved comment. Please check.

**Review by ajmaln — APPROVED** [review]

**Pull request merged**